lib/aquilon/aqdb/model/interface.py

```diff
@@ -249,7 +249,7 @@ class PublicInterface(Interface):
 
     extra_fields = ['bootable', 'port_group_name', 'port_group']
 
-    name_check = re.compile(r"^[a-z]+\d+[a-z]?$")
+    name_check = re.compile(r"^[a-z]+\d+[a-z]?\d?$")
```

Comment:
see https://github.com/quattor/configuration-modules-core/pull/1087/files#diff-90bc6b2e2a105ae5c62daf55ef95b2a7R137
(and there it is unittested)